### DISCUSSION TITLE
"Give item" button

### DISCUSSION THREAD

**Discussion by le-kote** [space-wizards/space-station-14#24723] [Feature Request]
Small addition that will stop this dropping items, instruments, food on floor to give it to someone. 
This would work like: You can offer an item, and player who near to you can accept or ignore it

**Comment by nymhole** [comment]
strip, hold item, click hand

**Reply by Everturning** [reply]
That's slow and not intuitive. The offer button would instantly transfer when clicked.

**Reply by NuclearCreeperIII** [reply]
Agreed, this functionality (as a single action, not fiddling with menus) is present on the rp-heavy ss13 server I currently play on, and sees a *lot* of use

**Reply by nymhole** [reply]
yea i also agree this current system sucks ass

**Reply by Boolean-Buckeye** [reply]
agree, ss13 had this, and the difference with strip is it isnt forceful
